demos/default/pages/env.tsx

```diff
@@ -1,19 +1,37 @@
 /**
  * https://nextjs.org/docs/basic-features/environment-variables
  */
-function EnvTest() {
+function EnvTest({ time }) {
   return (
     <div>
       <h1>Environment Variables</h1>
       <a href="https://nextjs.org/docs/basic-features/environment-variables">Read Docs</a>
-      <p>By default environment variables are only available in the Node.js environment, meaning they won&apos;t be exposed to the browser.</p>
       <p>
-        <code>NEXT_PUBLIC_</code> environment variables are available in the browser, and can be used to configure the application.
+        By default environment variables are only available in the Node.js environment, meaning they won&apos;t be
+        exposed to the browser.
       </p>
-      <p>✅ Public Environment token found: <code>{process.env.NEXT_PUBLIC_GREETINGS || 'NOT FOUND (something went wrong)'}</code></p>
-      <p>❌ Private Environment token should not be found:   <code>{process.env.TEST_ENV_VAR || 'Everything worked'}</code></p>
+      <p>
+        <code>NEXT_PUBLIC_</code> environment variables are available in the browser, and can be used to configure the
+        application.
+      </p>
+      <p>
+        ✅ Public Environment token found:{' '}
+        <code>{process.env.NEXT_PUBLIC_GREETINGS || 'NOT FOUND (something went wrong)'}</code>
+      </p>
+      <p>
+        ❌ Private Environment token should not be found: <code>{process.env.TEST_ENV_VAR ?? 'Everything worked'}</code>
+      </p>
+      <p>Rendered at {time}</p>
     </div>
   )
 }
 
-export default EnvTest
\ No newline at end of file
+export const getServerSideProps = async ({ params }) => {
```

Comment:
When this was statically rendered it was causing errors, because private env vars _are_ available on the server, so the HTML was mismatched. By SSRing it, we ensure that it's not included.